### PR TITLE
Adding FAQ

### PR DESCRIPTION
**Pull request by moovida**
This PR adds the FAQ page, as discussed in issue #122 

An html formatting happened at some point in the index.html file, if that is not wanted, I can remove it.

### CODE REVIEW

**Comment by dr-jts** [comment]
@lossyrob Does this need a signed ECA?  Or is it considered "minor"?

File: faq.html [hunk 1]
```diff
@@ -0,0 +1,453 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Frameset//EN" "http://www.w3.org/TR/html4/frameset.dtd">
+<html>
+  <head>
+    <meta content="text/html; charset=windows-1252" http-equiv="content-type">
+    <link rel="stylesheet" type="text/css" href="jts-doc.css">
+    <title> JTS | Documentation </title>
```
Comment (dr-jts) [inline faq.html]: Should fix title to JTS | FAQ

**Comment by lossyrob** [comment]
@dr-jts this would require a CLA.

@moovida thanks for the contribution!

A couple of housekeeping things:

You'll need to electronically sign a CLA. The process for doing that is here: https://www.eclipse.org/legal/ECA.php (see the side bar).

Another requirement is that all commits have a signoff. See: https://wiki.eclipse.org/Development_Resources/Handling_Git_Contributions#Git

You'll have to amend your commits to include the sign-off. Let me know if you have trouble doing that - one way is to `git reset --soft HEAD~2` on this branch and recommit the changes with the `-S` flag.

**Comment by moovida** [comment]
Ok, I see. I have no problem in signing the CLA, but currently the page is not reachable. I will try again later/tomorrow and report back.

**Comment by lossyrob** [comment]
Interesting. The page is working for me...it tells you to go to https://accounts.eclipse.org/user/login?destination=user/eca and sign into your account, perhaps that page won't give you troubles if the ECA link is still out.

**Comment by moovida** [comment]
Yep, it works for me also now. Just signed. Will go for the signoff now...

**Comment by moovida** [comment]
Ugh, I think I did the signoff , but it still fails. Anyone an idea about the reason?

**Comment by lossyrob** [comment]
What you just did seems to have generated additional commits. It also doesn't include the signoff on the commit - you'd see it in the message (see the sign-off message in commit of this PR: https://github.com/locationtech/geotrellis/pull/2345)

There are a couple ways of fixing this (rebasing and amending commits, resetting the branch and recommitting), but the most straightforward way might simply be to create a new branch, and cherry pick the commits (the following assumes that `upstream` is `https://github.com/locationtech/jts`)

```bash
> git checkout master
> git checkout -b gh-pages2
> git reset --hard upstream/gh-pages
> git cherry-pick -s b5824ec
> git cherry-pick -s 711318a
> git push origin gh-pages-2
```

Now make a new PR that pushes your `gh-pages-2` branch to `locationtech:gh-pages`. Make sure to set the correct upstream branch you're making the PR against. Reference this PR in that one, and you should be set.

**Comment by moovida** [comment]
Thanks @lossyrob , that did it indeed. Not sure what I had messed up before. I close this PR now.

**Pull request closed without merging**